src/apps/solarflare/README.md

```diff
@@ -0,0 +1,29 @@
+# Solarflare Ethernet Controller Apps
+
+## Solarflare (apps.solarflare.solarflare)
+
+The `Solarflare` app drives one port of a Solarflare SFN7 Ethernet
```

Comment:
Multiple instances of the Solarflare app can be instantiated on the same PCI device. Packets received from the network will be dispatched between apps based on destination MAC address and VLAN.